### ISSUE TITLE
CSR location information not copying to the work orders 

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#4903]
> What application are you using?

Signs & Markings Operations

> Describe the problem.

Location information from service requests is not copying to the work orders that are created from the SRs.

> Is there anything else we should know?

I just sent an email with screenshots to John Clary and Diana Martin!

> Website Address

https://atd.knack.com/signs-markings#work-order-signs/?view_2565_filters=%5B%7B%22value%22%3A%22ISSUED%22%2C%22text%22%3A%22Issued%22%2C%22operator%22%3A%22is%22%2C%22field%22%3A%22field_3265%22%7D%5D&view_2565_page=1

Internet Browser: Chrome

> Requested By
Christina T.

Request ID: DTS21-101490

**Comment by johnclary** [comment]
@amenity this might be me as well. Am taking a look and let Christina know.

**Comment by johnclary** [comment]
It appears that the [`LOCATION_NAME`](https://builder.knack.com/atd/signs-markings#data/object_176/fields/field_3235) form rule stopped picking up the `LOCATION_NAME_CSR` value.  I have no idea why this stopped working. The field rule has not been changed—it matches what's in the September test instance. I went ahead and inserted a new rule in the first position that sets "LOCATION_NAME" to "LOCATION_NAME_CSR" if "LOCATION_NAME_CSR" is not blank. It appears to have solved the problem but I asked the SMB folks to know if this persists.

I also did a batch update on all `signs_work_orders` records to re-populate the location name for any that are missing. This will be complete in an hour or so.
